server/src/test/java/org/elasticsearch/search/aggregations/metrics/StatsIT.java

```diff
@@ -86,39 +83,12 @@ public void testEmptyAggregation() throws Exception {
         assertThat(Double.isNaN(stats.getAvg()), is(true));
     }
 
+    /** this test was moved to {@link StatsAggregatorTests#testUnmapped()} */
```

Comment:
Hmm, how about making these non-abstract in `AbstractNumericTestCase`, that way we can just delete them entirely?  I don't think we'll be extending `AbstractNumericTestCase` anymore in the future and it will eventually go away, so probably ok to have concrete "no-op" implementations to make the transition time cleaner.

WDYT?

Reply:
I like it 👍